Library/MapleBacon/MapleBacon/ImageGifExtension.swift

```diff
@@ -7,35 +7,35 @@ import ImageIO
 
 extension UIImage {
 
-    class func imageWithCachedData(data: NSData) -> UIImage? {
-        guard data.length > 0 else { return nil }
+    class func image(withCachedData data: Data) -> UIImage? {
+        guard data.count > 0 else { return nil }
         
-        return isAnimatedImage(data) ? animatedImageWithData(data) : UIImage(data: data)
+        return isAnimatedImage(data: data) ? animatedImageWithData(data: data) : UIImage(data: data as Data)
```

Comment:
yep, actually it was like most of the other conversions done by the auto converter tool.
this is what I meant with "renaming some private methods" in the todo list